src/gui/systray.cpp

```diff
@@ -105,6 +105,11 @@ void Systray::create()
     }
     hideWindow();
     emit activated(QSystemTrayIcon::ActivationReason::Unknown);
+    foreach (Folder *f, FolderMan::instance()->map()) {
```

Comment:
Please use a for range loop, foreach is a deprecated "Qtism" (and we can expect that macro to disappear post-5.15).
Another option is to use std::any_of.
In both cases it's likely a good idea to hold the result of the `->map()` call in a `const auto` variable.

Reply:
You're right, thanks! Also makes a point for the 80-100 foreach uses in the rest of the code base, glimpsing through that lets assume that some deep copies could be saved -> #2219